_posts/2015-09-05-neo-hpsrt-jekyll-template.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Neo-HPSTR Jekyll Blog template
+date: 2015-09-05 12:00:00
+homepage: http://github.com/aron-bordin/neo-hpstr-jekyll-theme
+download: http://github.com/aron-bordin/neo-hpstr-jekyll-theme/archive/master.zip
+demo: http://aron-bordin.github.io/neo-hpstr-jekyll-theme/
+author: Aron Bordin
+thumbnail: neo-hpstr.png
+license: MIT
```

Comment:
Hello! Please change the license string from `MIT` to `MIT License` so the formatting will be similar to all other themes.